examples/plot_tikhonov.py

```diff
@@ -0,0 +1,247 @@
+            (probe_row - fix_row) < n_shape / 2 and
+            (probe_col - fix_col) > -n_shape / 2 and
+            (probe_col - fix_col) < n_shape / 2):
+
+            # Get probe timestamp relative to fixation
+            probe_time = probes_df.loc[probe_id]['t_probe']
+            probe_bin = np.where(bin_template < (probe_time - fixation_time))[0][-1]
+
+            # Define an image based on the relative locations
+            img = np.zeros(shape=(n_shape, n_shape))
+            row = -np.round(probe_row - fix_row) + n_shape/2 - 1
+            col = np.round(probe_col - fix_col) + n_shape/2 - 1
+            img[row, col] = 1
+
+            # Compute projection
+            basis_projection = strf_model.project_to_spatial_basis(img, spatial_basis)
+            this_fixation_spatial_features[probe_bin, :] = basis_projection
+
+    # Count spikes in window of interest relative to fixation
+    bins = fixation_time + bin_template
+    searchsorted_idx = np.searchsorted(spiketimes,
+                                       [fixation_time + window[0],
+                                        fixation_time + window[1]])
+    this_fixation_spike_counts = np.histogram(spiketimes[searchsorted_idx[0]:
+                                                         searchsorted_idx[1]], bins)[0]
+
+    # Accumulate
+    fixation_id = np.concatenate((fixation_id, this_fixation_id), axis=0)
+    spatial_features = np.concatenate((spatial_features, this_fixation_spatial_features), axis=0)
+    spike_counts = np.concatenate((spike_counts, this_fixation_spike_counts), axis=0)
+
+    # Zero pad
+    spatial_features = np.concatenate((spatial_features, np.zeros((n_zero_bins, n_spatial_basis))), axis=0)
+    fixation_id = np.concatenate((fixation_id, -999. * np.ones(n_zero_bins)), axis=0)
+
+# Convolve with temporal basis
+features = strf_model.convolve_with_temporal_basis(spatial_features, temporal_basis)
+
+# Remove zeropad
+features = features[fixation_id != -999.]
+
+########################################################
+# Visualize the distribution of spike counts
+
+########################################################
+
+plt.hist(spike_counts, 10)
+plt.show()
+
+########################################################
+# Plot a few rows of the design matrix
+
+########################################################
+
+plt.imshow(features[30:150,:], interpolation='none')
+plt.show()
+
+#################################################################
+# Design prior covariance matrix for Tikhonov regularization
+
+#################################################################
+
+prior_cov = strf_model.design_prior_covariance(sigma_temporal=3., sigma_spatial=5.)
+
+plt.imshow(prior_cov, cmap='Greys', interpolation='none')
+plt.colorbar()
+plt.show()
+
+np.shape(prior_cov)
+
+########################################################
+# Fit models
+
+########################################################
```

Comment:
I recently discovered that you don't need this line of `#####`. Perhaps worth removing?